handlers/location.js

```diff
@@ -0,0 +1,122 @@
+import { byCities, byStudios, byZipCodes } from '../data/locationMappings';
+import { escapeHTML, trackLink } from '../lib/util';
+import getFaq from '../lib/faq';
+
+import request from 'request-promise-native';
+import csvtojson from 'csvtojson';
+
+const uri = 'https://covid19nrw.netlify.com/.netlify/functions/get_nrw';
+
+export const handleLocation = async (ctx) => {
+    const location = ctx.dialogflowParams.location.structValue.fields;
+    console.log(`Detected location: ${location}`);
+    let messageText = ctx.dialogflowResponse;
+    const zipCode = location['zip-code'].stringValue;
+    let city = location.city.stringValue;
+    console.log(`zipCode: ${zipCode}`);
+    console.log(`city: ${zipCode}`);
+    if (byZipCodes[zipCode]) {
+        city = byZipCodes[zipCode].city;
+    }
+    if (byCities[city]) {
+        return handleCity(ctx, byCities[city]);
```

Comment:
you are returning here before the tracking could happen. on purpose?

Reply:
copy-paste error - fixed